crates/bevy_window/src/window.rs

```diff
@@ -602,12 +602,26 @@ impl Window {
     }
 }
 
+/// Describes the information needed for a window creation like width, height, title, present mode, etc.
+/// This should be set up before adding the [`WindowPlugin`]. Most of these settings can also later
+/// be configured through the [`Window`] resource.
+///
+/// See `examples/window/window_settings.rs` for usage.
```

Comment:
```suggestion
/// See [`examples/window/window_settings.rs`](https://github.com/bevyengine/bevy/blob/main/examples/window/window_settings.rs) for usage.
```

Reply:
I think the best format for this link is reference-style, due to the length of the URL.